modules/apps/dynamic-data-mapping/dynamic-data-mapping-validator/src/main/java/com/liferay/dynamic/data/mapping/validator/internal/DDMFormValidatorImpl.java

```diff
@@ -322,6 +342,21 @@ private void _validateDDMFormFieldPropertyValue(
 		}
 	}
 
+	private void _validateDDMFormFieldReferences(
+			List<DDMFormField> ddmFormFields)
+		throws DDMFormValidationException {
+
+		Set<String> duplicatedDDMFormFieldReferences = new HashSet<>();
+
+		_findDuplicatedDDMFormFieldReferences(
+			ddmFormFields, new HashSet<>(), duplicatedDDMFormFieldReferences);
+
+		if (SetUtil.isNotEmpty(duplicatedDDMFormFieldReferences)) {
+			throw new MustNotDuplicateFieldReference(
+				duplicatedDDMFormFieldReferences);
+		}
```

Comment:
@devpedrotavares since we're only calling this once, we can move this code to the `validate` method (right after the `_validateDDMFormFieldNames` call).

Reply:
Alright. Done!